install.sh

```diff
@@ -118,9 +118,14 @@ if [ $CI ]; then
 else
   read -p "Do you want to load example data (y/N)?" CONT
   if [ "$CONT" = "y" ]; then
+    echo "Downloading example data from Dispatch repository..."
     curl -O https://raw.githubusercontent.com/Netflix/dispatch/master/data/dispatch-sample-data.dump
     export PGPASSWORD='dispatch'
+    echo "Dropping database dispatch if it already exists..."
```

Comment:
I think we should warning the user before we drop the database as that is a destructive action (instead of being additive).